### PR TITLE
[Attu Metrics] Healthy Status View

### PR DESCRIPTION
**Pull request by alwayslove2013**

### CODE REVIEW

File: client/src/context/Prometheus.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            window.localStorage.setItem(
+              LAST_TIME_PROMETHEUS_INSTANCE,
+              prometheusInstance
+            );
+            window.localStorage.setItem(
+              LAST_TIME_PROMETHEUS_NAMESPACE,
+              prometheusNamespace
+            );
+            // openSnackBar(prometheusTrans('ready'));
+          } else openSnackBar(prometheusTrans('invalid'), 'error');
```
Comment (nameczz) [inline client/src/context/Prometheus.tsx:86]: need {} after else

File: client/src/context/Prometheus.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            );
+            window.localStorage.setItem(
+              LAST_TIME_PROMETHEUS_INSTANCE,
+              prometheusInstance
+            );
+            window.localStorage.setItem(
+              LAST_TIME_PROMETHEUS_NAMESPACE,
+              prometheusNamespace
+            );
+            // openSnackBar(prometheusTrans('ready'));
```
Comment (nameczz) [inline client/src/context/Prometheus.tsx:85]: do we need it?

File: client/src/pages/systemHealthy/SystemHealthyView.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+            {
+              label: 'Total Count',
+              data: prometheusData.totalVectorsCount,
+            },
+            {
+              label: 'Search Count',
+              data: prometheusData.searchVectorsCount,
+            },
+            {
+              label: 'Search Latency',
```
Comment (nameczz) [inline client/src/pages/systemHealthy/SystemHealthyView.tsx:142]: i18n?

File: client/src/utils/Format.ts [hunk 4]
```diff
@@ -148,6 +148,18 @@ export const formatAddress = (address: string): string => {
   return ip.includes(':') ? ip : `${ip}:${DEFAULT_MILVUS_PORT}`;
 };
 
+// format the prometheus address
+export const formatPrometheusAddress = (address: string): string => {
+  let formatAddress = address;
+  // add protocal (default http)
+  const withProtocol = address.includes('http');
+  if (!withProtocol) formatAddress = 'http://' + formatAddress;
```
Comment (nameczz) [inline client/src/utils/Format.ts:156]: how about https:// ?

File: server/src/prometheus/prometheus.service.ts [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+    prometheusAddress: string,
+    prometheusInstance: string,
+    prometheusNamespace: string
+  ) {
+    const addressValid = await http
+      .get(`${prometheusAddress}/-/ready`)
+      .then(res => res?.status === 200)
+      .catch(err => {
+        console.log(err);
+        return false;
```
Comment (nameczz) [inline server/src/prometheus/prometheus.service.ts:95]: why not throw error?

**Review by nameczz — CHANGES_REQUESTED** [review]

**Review by Tumao727 — APPROVED** [review]

File: client/src/context/Prometheus.tsx (hunk 2 above)
Comment (alwayslove2013) [reply client/src/context/Prometheus.tsx:85]: remove it

File: client/src/utils/Format.ts (hunk 4 above)
Comment (alwayslove2013) [reply client/src/utils/Format.ts:156]: 'https' also includes 'http'

File: server/src/prometheus/prometheus.service.ts (hunk 5 above)
Comment (alwayslove2013) [reply server/src/prometheus/prometheus.service.ts]: the error will be thrown now.

File: client/src/pages/systemHealthy/SystemHealthyView.tsx (hunk 3 above)
Comment (alwayslove2013) [reply client/src/pages/systemHealthy/SystemHealthyView.tsx]: use i18n now

**Review by shanghaikid — APPROVED** [review]

**Pull request merged**